app/wp-content/themes/kstg/library/scss/layout/_vehicle.scss

```diff
@@ -1,16 +1,24 @@
 .vehicle {
-  padding: 3.3%;
+  padding: 0 5% 80px;
   background-color: $light-grey;
-  h2 {
-    font-size: 1.6em;
-    margin: 25px 0;
+  clear: both;
+  .container {
```

Comment:
With the exception of padding I think all pages use this so, yes I think is the answer... would a mixin work for this? @derrickshowers